Tests/Opc.Ua.Security.Certificates.Tests/CertificateTestsForECDsa.cs

```diff
@@ -437,6 +424,23 @@ private static byte[] GetPublicKey(ECDsa ecdsa)
         }
         #endregion
 
+        #region Public static
+        public static ECCurveHashPair[] GetECCurveHashPairs()
```

Comment:
please use the runtime detection as in the develop/main374 branch, there are some linux distri which do not support brainpool